packages/icon/package.py

```diff
@@ -36,9 +36,6 @@ class Icon(Package):
     variant('host', default='daint', description='Build on described host (e.g daint)', multi=False, values=('tsa', 'daint'))
```

Comment:
should we not have tsa has default or maybe even no default ?

Reply:
hmm as you prefer, I thought at the time that Daint was the default for OpenACC people but know that has indeed changed :)